cms/templatetags/cms_tags.py

```diff
@@ -443,3 +443,9 @@ def get_context(self, context):
         context['CMS_TOOLBAR_CONFIG'] = context['request'].toolbar.as_json(context)
         return context
 register.tag(CMSToolbar)
+
+
+def get_by_key(mapping, key):
+    """ Get a value by key from a mapping. """
+    return mapping.get(key, None)
+register.filter(get_by_key)
```

Comment:
What's the use case here? Django's built-in template language will do a dictionary look up if you write `{{ dict_var.key }}`. Do you need to do a look up of a name that varies?